requirements.txt

```diff
@@ -6,25 +6,53 @@
 #
```

Comment:
As you removed some important dependencies, especially for the bot from the Pipfile, this of course needs to be adjusted as well